src/Collectors/TimelineDataCollector.php

```diff
@@ -86,9 +86,9 @@ public function addMeasure(
         string $label,
         float $start,
         float $end,
-        string $type = 'request',
-        string $action = 'request',
-        array $context = []
+        ?string $type = 'request',
```

Comment:
Why make it nullable? What should happen if we explicitly pass `null` as a param (aka missing test case)?

Reply:
I actually had to make it nullable because tests where failing 😀

When starting a measure and providing only a name, `$action` and `$label` properties will be null. When stopping the measure, we call `addMeasure` method with `null` values, while the method expect strings. Removing it will make tests fail:

```
Test  tests/Collectors/TimelineDataCollectorTest.php:testStartMeasureDuplication

  [TypeError] Argument 5 passed to AG\ElasticApmLaravel\Collectors\TimelineDataCollector::addMeasure() must be of the type string, null given, called in /home/vagrant/elastic-apm-laravel/src/Collectors/TimelineDataCollector.php on line 73
```

Reply:
But `$request` has always a default value, no need to make it nullable.

Reply:
oh... I see it now.